remoting/subsystem/src/main/java/org/jboss/as/remoting/RemotingSubsystem60Parser.java

```diff
@@ -0,0 +1,174 @@
+/*
+ * JBoss, Home of Professional Open Source.
+ * Copyright 2015 Red Hat, Inc., and individual contributors
```

Comment:
Fix year. We can't apply licence to the past.

Reply:
fixed